lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -520,6 +521,8 @@ def post(self, request):
                 except Exception as e: # pragma: no cover
                     log.error(e.message)
 
+                # delete prerequisites that are based on deleted course
+                remove_course_references(course.location.course_key)
```

Comment:
Do we interested to add some notes to the logs?

Reply:
We can't find out is something really removing here - it's just kind of request "remove if something exists"

Reply:
Yeap. So we can log something valuable inside the `remove_course_references`.
Just need to define the level.

Reply:
The place where we can detect actual deleting without extra requests is in external package - https://github.com/edx/edx-milestones/blob/master/milestones/data.py#L484
Do we really need to make changes there?

Reply:
I would add logs here with the message about our intention